6.0/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.0
+FROM node:6.2
```

Comment:
I'm not sure how our build process works (but I bet @tecnobrat can chime in on that), but to make this effectively "latest 6", it should be `FROM node:6`.

Also, as currently written, I think it would make our `articulate-node:6.0` actually load `6.2` which would be unexpected.  You might need to change the folder name to match to get it to work, [similar to this](https://github.com/nodejs/docker-node/commit/6a984d5b1f27b0344cbbab20019e77e6c0420b91)?  (Again, I defer to @tecnobrat.)